tests/integration/actions/exec/base.py

```diff
@@ -0,0 +1,104 @@
+"""The base class for exec interactive and stdout tests."""
+
+import difflib
+import json
+import os
+
+from typing import Generator
+from typing import Union
+
+import pytest
+
+from ..._common import fixture_path_from_request
+from ..._common import update_fixtures
+from ..._tmux_session import TmuxSession
+from ..._interactions import SearchFor
+from ..._interactions import Step
+from ....defaults import FIXTURES_DIR
+
+TEST_FIXTURE_DIR = os.path.join(FIXTURES_DIR, "integration", "actions", "exec")
+TEST_CONFIG_FILE = os.path.join(TEST_FIXTURE_DIR, "ansible-navigator.yaml")
```

Comment:
This is probably out of the scope, but it'd be truly great to use `pathlib` for any interactions with file paths and only convert them to `str` where really needed.

Reply:
yeah, I think there is probably many more place to make the switch to pathlib, but why not start now right?